### PR TITLE
Documentation for setup and sample system

### PR DESCRIPTION
**Pull request by kbalog**
closing #60

### CODE REVIEW

File: sample/README.md [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-- Next the system queries the arXivDigest API for user information. This is the information which the system bases its personalized recommendations on.
-- The system then creates personalized recommendations for each user, currently the recommendations are created by searching the Elasticsearch index with the users keywords for the best matching articles. 
-- Finally the articles found for each user are submitted to the arXivDigest API as recommendations. 
-
-## Usage
+  * First, it makes sure an Elasticsearch index with the correct mappings exist.  If not it will be created.
+  * Then, it queries the arXivDigest API for articles to add to the index.  These are the articles that will serve as candidate recommendations later.
+  * Next, the system queries the arXivDigest API for user information.  This is the information that the system bases its personalized recommendations on.
+  * The system then creates personalized recommendations for each user.  Specifically, it searches the Elasticsearch index for the best matching articlesm using the user's topics as queries.  Then, it aggregates, for each article, the relevance scores of all topics.
```
Comment (oyvi96) [inline sample/README.md]: it searches the Elasticsearch index for the best matching **articlesm** using the user's

Typo

File: Setup.md [hunk 2]
```diff
@@ -1,89 +1,69 @@
...
+  0. Prerequisites:
+    * [Python 3.6+](https://www.python.org/downloads/)
+    * [MySQL server](https://www.mysql.com/)
+  1. Clone this repository to a location of your choice (will be referred to as `REPO_PATH` below).
+  2. Execute all SQL scripts under [db/](db/) in sequential order, starting with [db/database.sql](db/database.sql) then V1, V2, etc.
+  3. Run `pip install .` while inside `REPO_PATH` to install the `arxivdigest` Python package and its dependencies.
+    * If installing with the purpose of development, use the command `pip install -e .` instead, to allow editing of the installed package.
+    * If running the service under an Apache Web Server, you may need to grant access to the respective user (e.g., www-data on Ubuntu) to the installed package.
+  4. Make sure to put [config.json](/config.json) in any of the below directories and update the settings specific to your system:
+    * `~/arxivdigest/config.json`
```
Comment (oyvi96) [inline Setup.md]: Lists are not rendered properly on github

File: Setup.md [hunk 3]
```diff
@@ -1,89 +1,69 @@
...
+## Initial setup
 
-#### Development
+  0. Prerequisites:
+    * [Python 3.6+](https://www.python.org/downloads/)
+    * [MySQL server](https://www.mysql.com/)
+  1. Clone this repository to a location of your choice (will be referred to as `REPO_PATH` below).
+  2. Execute all SQL scripts under [db/](db/) in sequential order, starting with [db/database.sql](db/database.sql) then V1, V2, etc.
+  3. Run `pip install .` while inside `REPO_PATH` to install the `arxivdigest` Python package and its dependencies.
+    * If installing with the purpose of development, use the command `pip install -e .` instead, to allow editing of the installed package.
```
Comment (oyvi96) [inline Setup.md]: Lists are not rendered properly on github

File: Setup.md [hunk 4]
```diff
@@ -1,89 +1,69 @@
...
+This document contains instructions on how to install and deploy the arXivDigest service.  Information on the sample recommender system can be found in a [separate document](sample/).
 
-Remember to configure the settings in [config.json](config.json) especially the secret_keys. More info at [Frontend configuration](arxivdigest/frontend/README.md#configurations) and  [API configuration](arxivdigest/api/README.md#configurations)
 
-Static files should be served by a webserver for best performance, to achieve this a 'data_path' must be configured in the config. Then the webserver needs to be configured to reroute calls to '/static' to the folder named static that gets generated inside this location after the first launch. If not rerouted these files will be served through Flask.
+## Initial setup
 
-#### Development
+  0. Prerequisites:
+    * [Python 3.6+](https://www.python.org/downloads/)
```
Comment (oyvi96) [inline Setup.md]: Lists are not rendered properly on github.

File: Setup.md [hunk 5]
```diff
@@ -1,89 +1,69 @@
...
-The API and frontend should be started by running 'app.py' in their respective folder while developing.
 
-Make sure that port 80 is free for the frontend and 5000 is free for the API or configure the frontend and api dev_ports in [config.json](/config.json)
+## Installing updates
 
-### Core
+  1. Pull changes from this repository.
+  2. Execute any new SQL scripts in [db/](db/).
+  3. Run `pip install .` while inside `REPO_PATH` to update the package and its dependencies.
+    * If needed, check permissions for the installed package.
```
Comment (oyvi96) [inline Setup.md]: Lists are not rendered properly on github.

File: sample/README.md (hunk 1 above)
Comment (kbalog) [reply sample/README.md]: Fixed

File: Setup.md (hunk 2 above)
Comment (kbalog) [reply Setup.md]: Fixed

File: Setup.md (hunk 3 above)
Comment (kbalog) [reply Setup.md]: Fixed

File: Setup.md (hunk 4 above)
Comment (kbalog) [reply Setup.md]: Fixed

File: Setup.md (hunk 5 above)
Comment (kbalog) [reply Setup.md]: Fixed

**Pull request merged**